opencraft/settings.py

```diff
@@ -411,3 +415,7 @@
 
 # Limit the number of log entries fetched for each instance, for performance
 LOG_LIMIT = env.int('LOG_LIMIT', default=10000)
+
+# Subdomain blacklist #########################################################
+
+SUBDOMAIN_BLACKLIST = env.list('SUBDOMAIN_BLACKLIST', default=[])
```

Comment:
@antoviaque No.  How about this list I just made up?

    www, mail, blog, web, ftp, ns1, ns2, smtp, test, opencraft, api, forum, admin

(Note that names shorter than three characters are disallowed anyway.)